platform/iphone/export/export.cpp

```diff
@@ -114,54 +114,14 @@ class EditorExportPlatformIOS : public EditorExportPlatform {
 			return false;
 		}
 
-		int segments = 0;
-		bool first = true;
 		for (int i = 0; i < pname.length(); i++) {
 			CharType c = pname[i];
-			if (first && c == '.') {
-				if (r_error) {
-					*r_error = TTR("Identifier segments must be of non-zero length.");
-				}
-				return false;
-			}
-			if (c == '.') {
-				segments++;
-				first = true;
-				continue;
-			}
-			if (!((c >= 'a' && c <= 'z') || (c >= 'A' && c <= 'Z') || (c >= '0' && c <= '9') || c == '-')) {
+			if (!((c >= 'a' && c <= 'z') || (c >= 'A' && c <= 'Z') || (c >= '0' && c <= '9') || c == '-') || c == '.') {
```

Comment:
Won't this prevent using `.`, and thus reverse DNS identifiers?

Reply:
🤦‍♂ fixed..

Reply:
Hm, it still seems to be the same to me. In pseudo code it's `if (!(is_alphanumeric()) || c == '.') error`.

The `|| c == '.'` should be in the `!(  )`.

Reply:
I don't know why but it seemed like git didn't added the changes (or maybe I just forget it), it's correctly now.